engine/runtime-integration-tests/src/test/java/org/enso/logging/service/logback/telemetry/test/TestLogFormatter.java

```diff
@@ -0,0 +1,124 @@
+package org.enso.logging.service.logback.telemetry.test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.nullValue;
+
+import org.enso.logging.service.logback.telemetry.LogFormatter;
+import org.enso.logging.service.logback.telemetry.LogMessage;
+import org.junit.Test;
+
+public class TestLogFormatter {
+  @Test
+  public void shouldNotTransformLog_WithIncorrectMessage() {
+    var logMessage = createLogMessage("Message - arg=1", 1);
+    var json = LogFormatter.transform(logMessage);
+    assertThat("was not transformed", json, is(nullValue()));
+  }
+
+  @Test
+  public void shouldNotTransformLog_WithoutArguments() {
+    var logMessage = createLogMessage("Message: arg=1");
+    var json = LogFormatter.transform(logMessage);
+    assertThat("was not transformed", json, is(nullValue()));
+  }
+
+  @Test
+  public void shouldNotTransformLog_WithRestrictedMetadata() {
+    var logMessage = createLogMessage("Message: type=1", 1);
```

Comment:
Can we try to use different text than `Message`? For example one with some special characters?

Reply:
Different message with multiple words tested in https://github.com/enso-org/enso/blob/9c7f2b4cd0298afa57cf649f0dc77e248740c13b/engine/runtime-integration-tests/src/test/java/org/enso/logging/service/logback/telemetry/test/TestLogFormatter.java#L66-L71